pool-agent/cmd/config.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/lxc/lxd/shared/api"
+	"github.com/pelletier/go-toml/v2"
+)
+
+// Config is config map for pool agent.
+type Config struct {
+	ImageAlias          string              `toml:"image_alias"`
+	ResourceTypesMap    []ResourceTypesMap  `toml:"resource_types_map"`
+	ResourceTypesCounts ResourceTypesCounts `toml:"resource_types_counts"`
+}
+
+// ResourceTypesMap is resource configuration for pool mode.
+type ResourceTypesMap struct {
+	Name string `toml:"name"`
+
+	CPUCore int    `toml:"cpu"`
+	Memory  string `toml:"memory"`
+}
+
+// ResourceTypesCounts is counts for resouce types.
+type ResourceTypesCounts map[string]int
+
+// LoadConfig LoadConfig loads config from configPath
+func LoadConfig() (Config, error) {
+	f, err := os.ReadFile(configPath)
+	if err != nil {
+		return Config{}, fmt.Errorf("failed read config file: %w", err)
+	}
+	var s Config
+	if err := toml.Unmarshal(f, &s); err != nil {
+		return Config{}, fmt.Errorf("parse config file: %w", err)
+	}
+	return s, nil
+}
+
+// ParseImageAlias parses LXD image alias.
+func ParseImageAlias(s string) (api.InstanceSource, error) {
```

Comment:
`ParseImageAlias` is the same function in shoes-lxd-multi-server.
https://github.com/whywaita/shoes-lxd-multi/blob/58c0274362ad68916b3afcc2015a6f8e08ac69e0/server/pkg/api/server_add_instance.go#L282

Could you update to export function in shoes-lxd-multi-server?
We won't maintenanse a same code in other functions.